packages/components/src/components/TextInput/TextInput.tsx

```diff
@@ -124,7 +122,7 @@ export const TextFieldRoot = (
     const wasClicked = useRef(false);
 
     return (
-        <div className={cn(styles.root, className)} data-status={status} data-test-id={dataTestId}>
```

Comment:
`join().filter(Boolean).join(' ')` in case `className` is undefined, else we will have `class="root_hash undefined`"`